app/Http/Controllers/BackendController.php

```diff
@@ -0,0 +1,192 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Input;
+use Validator;
+use Redirect;
+use App\Models\CovidEntry;
+use App\Models\CovidList;
+use App\Models\CovidCase;
+use DB;
+
+class BackendController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        //
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        //
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        $store = new CovidList();
+
+        $data=Input::except(array('_token'));
+        
+        $rule = array(
```

Comment:
Pls use `Form Request` instead of direct validation.

Reply:
Yes, I change it again.